.github/workflows/deploy-docker.yaml

```diff
@@ -109,48 +119,30 @@ jobs:
         with:
           context: .
           file: apps/${{ matrix.packages.name }}/Dockerfile
-          tags: |
-            ${{ inputs.container-registry }}/${{ inputs.image-prefix }}/${{ matrix.packages.name }}:${{ matrix.packages.imageTag }}
+          tags: ${{ inputs.container-registry }}/${{ inputs.image-prefix }}/${{ matrix.packages.name }}:${{ matrix.packages.imageTag }}
           push: ${{ inputs.push }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
```

Comment:
Why not just use gha cache?

Reply:
Link?

Reply:
Just use `type=gha`, and GitHub Actions will handle everything automatically.

https://docs.docker.com/build/cache/backends/gha/

Reply:
Also don't forget to add `mode=max` to `cache-to` to cache all layers.